examples/player/simid_player.js

```diff
@@ -131,6 +131,30 @@ class SimidPlayer {
     });
   }
 
+  /**
+   * Player closes the ad and SIMID creative.
+   */
+  playerCloseAd() {
```

Comment:
change to closeAd. When someone calls this it will be
player.closeAd()

no need to have it be
player.playerCloseAd()

Same with the other functions.

Or better yet, just get rid of this function entirely and call stopAd directly from wherever you are calling this.

Reply:
got rid of them